<?xml version="1.0" ?>
<!--joy red mode-->
<launch>
	<node pkg="robot_joy" type="robot_joy_node" name="robot_joy" >
	<param name="axis_linear_x" value="1" type="int" />
	<param name="axis_linear_y" value="0" type="int" />
	<param name="axis_angular" value="3" type="int" />
	<param name="axis_accelerator" value="5" type="int" />
	<param name="axis_deceleration" value="2" type="int" />
	<param name="button_max_linear_increase" value="0" type="int" />
	<param name="button_max_linear_reduce" value="1" type="int" />
	<param name="button_max_angular_increase" value="2" type="int" />
	<param name="button_max_angular_reduce" value="3" type="int" />

	<param name="linear_vel_max" value="0.7" type="double" />
	<param name="angular_vel_max" value="1.0" type="double" />
	<param name="linear_vel" value="0.3" type="double" />
	<param name="angular_vel" value="0.5" type="double" />
	</node>
	<node respawn="true" pkg="joy" type="joy_node" name="joy_node">
	<param name="dev" type="string" value="/dev/input/js0" />
    	<param name="deadzone" value="0.12" />
	</node>
</launch>
